Controls/main.c

```diff
@@ -0,0 +1,17 @@
+#include "MotorController.h"
+#include "stdlib.h"
+#include <time.h>
+
+void delay(int seconds){
+    clock_t start_time = clock();
+    while ((clock()/(CLOCKS_PER_SEC))<start_time+seconds){
+        ;
+    }
+}
+
+void main(){
+    while(1){
+        delay(.250);
```

Comment:
.250 when getting cast to an int is 0, so the function will be called to delay for 0 seconds. Types in C are very strict. So the ideal way to get around this would be to accept the parameter in milliseconds and put a unit conversion on line 7.